hubble/client/client.py

```diff
@@ -54,15 +55,29 @@ def delete_personal_access_token(self, name: str) -> Union[requests.Response, di
             data={'name': name},
         )
 
-    def get_user_info(self, log_error: bool = True) -> Union[requests.Response, dict]:
+    def get_user_info(self, log_error: bool = True) -> dict:
         """Get current logged in user information.
 
-        :returns: `requests.Response` object as returned value
-            or indented json if jsonify.
+        :returns: dict user information.
         """
-        return self.handle_request(
+
+        try:
+            decoded = validate_jwt(self._token)
+            if isinstance(decoded, dict) and decoded.get('user'):
+                user = decoded.get('user')
+                if user.get('status') == 'deletion-in-progress':
```

Comment:
Does it make sense to also return the user info directly when then the status is `active`?

Reply:
I agree. I don't have strong reference on it. Just wanted to make changes as close as possible with deletion.
Let me remove this.

Reply:
*not remove. Let me add `active`.